Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -1672,14 +1672,16 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     void dragEntered(WebCore::DragData&, const String& dragStorageName = String());
     void dragUpdated(WebCore::DragData&, const String& dragStorageName = String());
     void dragExited(WebCore::DragData&);
-    void performDragOperation(WebCore::DragData&, const String& dragStorageName, SandboxExtensionHandle&&, Vector<SandboxExtensionHandle>&&);
+    void performDragOperation(WebCore::DragData&, const String& dragStorageName, SandboxExtensionHandle&&, Vector<SandboxExtensionHandle>&&, const std::optional<WebCore::FrameIdentifier>& = std::nullopt);
```

Comment:
optional FrameIdentifier is passed in 2 registers.  There's no reason to have a const reference here.  Just pass by value.